mqtt/src/main/scala/akka/stream/alpakka/mqtt/settings.scala

```diff
@@ -290,23 +301,26 @@ final class MqttConnectionSettings private (val broker: String,
   def withDisconnectTimeout(disconnectTimeout: Int, unit: TimeUnit): MqttConnectionSettings =
     copy(disconnectTimeout = FiniteDuration(disconnectTimeout, unit))
 
-  private def copy(broker: String = broker,
-                   clientId: String = clientId,
-                   persistence: org.eclipse.paho.client.mqttv3.MqttClientPersistence = persistence,
-                   auth: Option[(String, String)] = auth,
-                   socketFactory: Option[javax.net.ssl.SSLSocketFactory] = socketFactory,
-                   cleanSession: Boolean = cleanSession,
-                   will: Option[MqttMessage] = will,
-                   automaticReconnect: Boolean = automaticReconnect,
-                   keepAliveInterval: FiniteDuration = keepAliveInterval,
-                   connectionTimeout: FiniteDuration = connectionTimeout,
-                   disconnectQuiesceTimeout: FiniteDuration = disconnectQuiesceTimeout,
-                   disconnectTimeout: FiniteDuration = disconnectTimeout,
-                   maxInFlight: Int = maxInFlight,
-                   mqttVersion: Int = mqttVersion,
-                   serverUris: immutable.Seq[String] = serverUris,
-                   sslHostnameVerifier: Option[javax.net.ssl.HostnameVerifier] = sslHostnameVerifier,
-                   sslProperties: Map[String, java.lang.String] = sslProperties): MqttConnectionSettings =
+  private def copy(
```

Comment:
You need to run `sbt test:compile` so all files as formatted correctly.

Reply:
It's really strange, I've invoked test task and automatically formatted by scalafmt.